kasa/cli.py

```diff
@@ -833,20 +833,20 @@ async def usage(dev: Device, year, month, erase):
 
     if erase:
         echo("Erasing usage statistics..")
-        return await usage.erase_stats()
+        return await usage.erase_stats()  # type: ignore[attr-defined]
```

Comment:
It'd be better to cast the module than adding type ignores for every call site.

Reply:
Done